pyros_schemas/ros/schemagic.py

```diff
@@ -51,7 +51,8 @@ def create(ros_msg_class,
 
     members_types = _get_rosmsg_fields_as_dict(ros_msg_class)
     members = {}
-    for s, stype in six.iteritems(members_types):
+    schema_instance = RosSchema()
```

Comment:
So you mean that instead of creating a child class (CustomMsgSchema) of RosSchema, and taking and instance of that class,
it is better to instantiate a RosSchema and usedirectly that instance ?
How about the second time we need that Schema, we have to create it again it seems ?

Reply:
I tried to solve your source as much as possible without modifying it if possible. So I did not try various methods. Sorry and thank you of your efforts.